lib/fastlane/actions/xcov.rb

```diff
@@ -0,0 +1,91 @@
+module Fastlane
+  module Actions
+    class XcovAction < Action
+      def self.run(values)
+        Actions.verify_gem!('xcov')
+        require 'xcov'
+
+        begin
+          Xcov::Manager.new.work(values)
+          true
+        end
+      end
+
+      def self.description
+        "Nice code coverage reports without hassle"
+      end
+
+      def self.details
+        "More information: https://github.com/nakiostudio/xcov"
+      end
+
+      def self.author
+        "nakiostudio"
+      end
+
+      def self.available_options
+        containing = FastlaneCore::Helper.fastlane_enabled? ? './fastlane' : '.'
```

Comment:
Instead of doing this, you should actually do it similar to `sigh`: 

```ruby
Actions.verify_gem!('xcov')
require 'xcov'
Xcov::Options.available_options
```